ci/fuzz/run.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/sh
+
+set -eu
+
+MAX_TOTAL_TIME=600
+
+# Build base Docker image.
+docker build -t spicy-fuzz .
+
+# Create fuzzing binaries.
+mkdir -p spicy-fuzz
+docker run -it --rm \
+    -v "${PWD}"/spicy-fuzz:/out \
+    -e OUT=/out \
+    -v "${PWD}"/../../:/work \
+    -e CXX=clang++-12 \
+    -e CC=clang-12 \
+    -e SANITIZER=address \
+    spicy-fuzz \
+    /work/ci/fuzz/build.sh
+
+# Run individual fuzzers.
+
+# shellcheck disable=SC2043
+for parser in Message; do
+mkdir -p spicy-fuzz/corpus-fuzz-dhcp-"$parser"
+docker run -it --rm \
+    -v "${PWD}"/spicy-fuzz:/work \
+    -e SPICY_FUZZ_PARSER="$parser" \
+    spicy-fuzz \
+    /work/fuzz-dhcp -max_total_time=$MAX_TOTAL_TIME -jobs="$(nproc)" /work/corpus-fuzz-dhcp-"$parser"
+done
```

Comment:
Any reason not to factor this out into a function instead of repeating it below?

Reply:
The way this is currently written we assume that fuzzers would be run sequentially which can take a long time. I wasn't sure whether that was the best way forward.

I'll look into how to create multidimensional arrays in here to capture data of the form `[GRAMMAR, [PARSER..]` we can iterate over.

Reply:
I meant just the bodies of the for loops; that's always the same, isn't it?